bentoml/_internal/cli/bento_management.py

```diff
@@ -170,35 +174,56 @@ def delete_target(target):
 
                 if delete_confirmed:
                     bento_store.delete(bento.tag)
-                    click.echo(f"{bento} deleted")
+                    logger.info(f"{bento} deleted")
 
         for target in delete_targets:
             delete_target(target)
 
-    @cli.command(help="Export Bento to a tar file")
+    @cli.command()
     @click.argument("bento_tag", type=click.STRING)
     @click.argument(
-        "out_file", type=click.File("wb"), default=sys.stdout, required=False
+        "out_path",
+        type=click.STRING,
+        default="",
+        required=False,
     )
-    def export(bento_tag, out_file):
-        """Export Bento files to a tar file
+    def export(bento_tag, out_path):
+        """Export Bento files to an archive file
+
+        \b
+        arguments:
+        BENTO_TAG: bento identifier
+        OUT_PATH: output path of exported bento.
+          If this argument is not provided, bento is exported to name-version.bento in the current directory.
+          Supported formats are tar ('tar'), tar.gz ('gz'), tar.xz ('xz'), tar.bz2 ('bz2'), and zip
 
-        bentoml export FraudDetector:latest > my_bento.tar
+        \b
+        examples:
+        bentoml export FraudDetector:20210709_DE14C9
         bentoml export FraudDetector:20210709_DE14C9 ./my_bento.tar
```

Comment:
@larme @sauyon do we allow user to provide a path/directory when exporting? 

e.g. `bentoml export s3://foo/bar/` which creates a file `s3://foo/bar/FraudDetector_20210709_DE14C9.bento` automatically?

Reply:
Yes, that should indeed work. Note that for the remote sources I didn't add a check that the remote was a directory, so I consciously chose not to document this because sometimes shells can be weird with including final `/`s with autocomplete and such.